include/Utils/ReplaceIRVar.h

```diff
@@ -0,0 +1,23 @@
+#ifndef REPLACEIRVAR_H
+#define REPLACEIRVAR_H
+
+#include "llvm/IR/Function.h"
+#include "llvm/IR/BasicBlock.h"
+#include "llvm/IR/Module.h"
+#include <iostream>
+#include <map>
+
+class ReplaceIRVar{
+private:
+	std::string NewName;
+	std::map<std::string,std::string> HashMap;
```

Comment:
Can you please add a comment about what are the keys and values in this HashMap?